src/infra/api-key/csv/reader.go

```diff
@@ -65,14 +61,8 @@ func (r *Reader) Load(_ context.Context) (map[string]*entities.UserClaims, error
 			return nil, fmt.Errorf("invalid number of cells, should be %d", csvRowLen)
 		}
 
-		r.hasher.Reset()
-		_, err = r.hasher.Write([]byte(cells[csvHashOffset]))
-		if err != nil {
-			return nil, fmt.Errorf("failed to hash api key")
-		}
-
-		apiKeyHash := base64.StdEncoding.EncodeToString(r.hasher.Sum(nil))
-		claims[apiKeyHash] = &entities.UserClaims{
+		apiKeySha256 := cells[csvAPIKeyOffset]
```

Comment:
Related to comment above that produces this confusion here

Reply:
done